dev/com.ibm.jbatch.container/src/com/ibm/jbatch/container/jsl/impl/StepModelSerializerImpl.java

```diff
@@ -23,43 +23,37 @@
 import javax.xml.bind.Marshaller;
 import javax.xml.namespace.QName;
 
-
-
-
 import com.ibm.jbatch.container.jsl.JSLValidationEventHandler;
 import com.ibm.jbatch.container.jsl.ModelSerializer;
-import com.ibm.jbatch.container.jsl.ValidatorHelper;
 import com.ibm.jbatch.jsl.model.Step;
 
 public class StepModelSerializerImpl implements ModelSerializer<Step> {
 
-	@Override
-	public String serializeModel(Step model) {
-		return marshalStep(model);
-	}
+    @Override
+    public String serializeModel(Step model) {
+        return marshalStep(model);
+    }
 
     private String marshalStep(Step step) {
-    	String resultXML = null;
-    	JSLValidationEventHandler handler = new JSLValidationEventHandler();
-    	try {
-    		JAXBContext ctx = JAXBContext.newInstance("com.ibm.jbatch.jsl.model");
-    		Marshaller m = ctx.createMarshaller();
-    		m.setSchema(ValidatorHelper.getXJCLSchema());
-    		m.setEventHandler(handler);
-    		ByteArrayOutputStream baos = new ByteArrayOutputStream();
-    		//m.marshal(job, baos);
-    		/*
-    		 * from scott: 
-    		 */
-    		m.marshal( new JAXBElement(
-    				new QName("http://xmlns.jcp.org/xml/ns/javaee","step"), Step.class, step ), baos);
-    		resultXML = baos.toString();
-    	}
-    	catch(Exception e){
-    		throw new RuntimeException("Exception while marshalling Step", e);
-    	}
-    	
-    	return resultXML;
+        String resultXML = null;
+        JSLValidationEventHandler handler = new JSLValidationEventHandler();
+        try {
+            ClassLoader currentClassLoader = Step.class.getClassLoader();
+            JAXBContext ctx = JAXBContext.newInstance("com.ibm.jbatch.jsl.model", currentClassLoader);
+            Marshaller m = ctx.createMarshaller();
+            m.setEventHandler(handler);
+            ByteArrayOutputStream baos = new ByteArrayOutputStream();
+            //m.marshal(job, baos);
+            /*
+             * from scott:
+             */
+            m.marshal(new JAXBElement(new QName("http://xmlns.jcp.org/xml/ns/javaee", "step"), Step.class, step), baos);
```

Comment:
Make up a NS for serialization:     
 `new QName("http://com.ibm.jbatch.model/serialization", "step"), `

It's probably best not to muddy the waters by using an element that looks like it's part of the spec XSD but really isn't.

Reply:
That was how I found the code, but I'll change it to the string above (also in the jobModel).